sdk/Oracle/azure-mgmt-oracledatabase/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
```

Comment:
@ChenxiJiang333 why is this in the "Oracle" directory? Generally the service directory matches the name of the package so oracledatabase in this case.

Reply:
Thanks for feedback and we will rename the folder soon.

Reply:
> @ChenxiJiang333 why is this in the "Oracle" directory? Generally the service directory matches the name of the package so oracledatabase in this case.

My fault for not checking the output folder config, has made a quick fix on it.